### ISSUE TITLE
String Length of exactly 4k characters fails with 'Invalid Precision Value'

### ISSUE THREAD

**Issue by bsEmo** [TimelordUK/node-sqlserver-v8#184]
When running a bulk insert through the table functionality with a column of varchar(max) and a string of exactly 4k characters the call will error with message "Invalid Precision Value". I have also been able to replicate in a simple fashion:

```js
"use strict";
const mssql = require("mssql/msnodesqlv8");

const STR_LEN = 4000;
const str = "1".repeat(STR_LEN);

const sqlConfig = {
  driver: "msnodesqlv8",
  server: "...",
  options: {
    trustedConnection: true,
    useUTC: true
  }
};
mssql.connect(sqlConfig).then(() => {
  return new mssql.Request()
    .input("str", mssql.VarChar(mssql.MAX), str)
    .query("DECLARE @s VARCHAR(MAX) = @str; SELECT @s AS s")
    .then((req) => {
      console.log(req);
    })
    .catch((err) => {
      console.error(err);
    });
});
```

```
Error: RequestError: Invalid precision value
    at handleError (C:\WIP\aa-tests\mssql-testing\node_modules\mssql\lib\msnodesqlv8\request.js:258:21)
    at StreamEvents.emit (events.js:198:13)
    at errors.forEach.err (C:\WIP\aa-tests\mssql-testing\node_modules\msnodesqlv8\lib\reader.js:37:20)
    at Array.forEach (<anonymous>)
    at routeStatementError (C:\WIP\aa-tests\mssql-testing\node_modules\msnodesqlv8\lib\reader.js:30:14)
    at invokeObject.end (C:\WIP\aa-tests\mssql-testing\node_modules\msnodesqlv8\lib\reader.js:209:11)
    at freeStatement (C:\WIP\aa-tests\mssql-testing\node_modules\msnodesqlv8\lib\driver.js:183:13)
    at cppDriver.freeStatement (C:\WIP\aa-tests\mssql-testing\node_modules\msnodesqlv8\lib\driver.js:163:13)
```

### Software versions

  * NodeJS: v10.16.0
  * node-mssql: 6.2.3
  * SQL Server: Microsoft SQL Server 2016 (SP2-CU11-GDR) (KB4535706) - 13.0.5622.0 (X64) 
	Dec 15 2019 08:03:11 
	Copyright (c) Microsoft Corporation
	Developer Edition (64-bit) on Windows Server 2012 R2 Standard 6.3 <X64> (Build 9600: ) (Hypervisor)

**Comment by TimelordUK** [comment]
thanks yes I can reproduce this issue, I have started to take a look at it will  keep you posted.

**Comment by TimelordUK** [comment]
this should now be fixed in 2.0.8

**Comment by bsEmo** [comment]
Testing checks out. Thanks for the quick turnaround!
